samples/Sentry.Samples.Maui/Sentry.Samples.Maui.csproj

```diff
@@ -6,7 +6,8 @@
       On Mac, we'll also build for iOS and MacCatalyst.
       On Windows, we'll also build for Windows 10.
     -->
-    <TargetFrameworks>net8.0-android</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworks);net8.0-android</TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net8.0-windows10.0.19041.0</TargetFrameworks>
     <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net8.0-windows10.0.19041.0;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
```

Comment:
Did you duplicate `TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))"` on purpose?

Reply:
No I think that must have happened accidentially during one of the merges from main... I'll remove the dupplicate. Thanks @bricefriha